artifacts/deployment/00-etcdproxy-controller.yaml

```diff
@@ -127,4 +127,4 @@ spec:
         command:
           - /etcdproxy-controller
           - "--etcd-core-url=https://etcd-svc-1.etcd.svc:2379"
-        imagePullPolicy: Always
+        imagePullPolicy: Always
```

Comment:
nit, i personally prefer the newline at the end.

Reply:
Fixed.